### ISSUE TITLE
Maxweight attribute unit consistency

### ISSUE THREAD

**Issue by JonusAngelus** [openstreetmap/iD#6707]
Hello !

The documentation for the attribute says that it is in [tonne] but the field has for title "Maximum Weight [kg].

When you type the value in [kg] it doesn't convert to [tonne] and updates the value in the attribute field (because I guess it's the same var).

Maybe update the field label or implement unit conversion to default metric unit [tonne] ?

Greetings

![image](https://user-images.githubusercontent.com/14310483/62289209-62dbac80-b45e-11e9-8872-22e2f66b4659.png)

**Comment by 1ec5** [comment]
This appears to be an error in the French translation of the string. You can help correct this or any other translated string [at Transifex](https://www.transifex.com/openstreetmap/id-editor/).

**Comment by eMerzh** [comment]
i've update the french translation, removing the unit note that was not in the original.

We probably can rename this issue to something like "add unit dropdown to maxweight field"

**Comment by quincylvania** [comment]
> We probably can rename this issue to something like "add unit dropdown to maxweight field"

@eMerzh Good suggestion. There are lots of other fields that could use unit dropdowns too, like `height`, `maxheight`, `ele`, `length`, `width`, `diameter`, `water_volume`. It'd be nice to come up with some easier system for these.

**Comment by bhousel** [comment]
yeah support for units has been on our radar for a while, see also 
https://github.com/openstreetmap/iD/issues/5658#issuecomment-450387386
#3614 
#3615
